### PR TITLE
Past_projects

### PR DESCRIPTION
**Pull request by rahmot**
The past projects will have to be formatted in HTML and CSS to add the boxes and designs.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
```
Comment (bebatut) [inline .gitpod.yml:1]: This file probably belongs to another PR

File: past_projects.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Past Projects
```
Comment (bebatut) [inline past_projects.md:1]: This page will be likely hidden at the beginning as we do not have any projects
Btw should it really "Past projects"? I have the feeling it would be more "Past Gmentees", that may grouped by topics

File: projects_contribution.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Galaxy Training Network
```
Comment (bebatut) [inline projects_contribution.md:1]: I do not understand the purpose of this page, compared to the one before.

File: past_projects.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Welcome to the GMN Past Projects, a group of GMN projects by past participants. 
+
+The areas of project are not limited to: Contributing to the community, Running data analysis, Instructing students, and Carrying out projects. 
+
+## Here’s what our participants worked on
+
+### [Galaxy Tools](#)
```
Comment (bebatut) [inline past_projects.md:9]: To what would that link?

**Review by bebatut — COMMENTED** [review]
Thanks. I added some questions. 
Also can you remove all files that do not belong to this PR?

File: .gitpod.yml (hunk 1 above)
Comment (rahmot) [reply .gitpod.yml:1]: Yes, this was used to configure Gitpod so that I can preview. How can I move it to another PR? Do I have to delete this first and then create it again?

File: past_projects.md (hunk 4 above)
Comment (rahmot) [reply past_projects.md:9]: The link was supposed to redirect us to the different topics Gmentees worked on. The "Galaxy Tools" is like the topics group name, that is, all topics under it are related to tools. But since we are opting for "Past Gmentees" that would not be necessary again. 

Now the question is, how do we automate the grouping of Gmentees by topics?

File: past_projects.md (hunk 2 above)
Comment (rahmot) [reply past_projects.md:1]: Okay! We can change it to Past Gmentees since we are using project again.

File: projects_contribution.md (hunk 3 above)
Comment (rahmot) [reply projects_contribution.md:1]: The "Galaxy Training Network" is like the "Galaxy Tools" (the one below). So on this page, we have a list of topic related GTN. But all that we change since we are opting for "Past Gmentees".

File: past_projects.md (hunk 4 above)
Comment (bebatut) [reply past_projects.md:9]: We could have an automated way to create this page.
For example, in OLS, all projects details (participants, mentor, description) are stored in YAML file. The content of the YAML file is imported in the template of the project page and is added to the final HTML page using loops.
We could think about having something similar, with one of the attributes being a topic. We could then in the template, parse the Gmentees and group them by topic.

File: projects_contribution.md (hunk 3 above)
Comment (bebatut) [reply projects_contribution.md:1]: I got that. I just do not understand why there is an extra page for that.

**Comment by rahmot** [comment]
We have started working on this in another PR, so I am closing this.
